tests/testthat/test-opart-poisson.R

```diff
@@ -0,0 +1,87 @@
+library(testthat)
```

Comment:
Hi @tdhock ,

Does these tests look fine to you ? I worked out an example for poisson loss with (1,2,3) as data.vec and 1 as penalty for testing correctness.